administrator/components/com_banners/helpers/html/banner.php

```diff
@@ -111,6 +111,6 @@ public static function pinned($value, $i, $enabled = true, $checkbox = 'cb')
 			),
 		);
 
-		return JHtml::_('jgrid.state', $states, $value, $i, 'banners.', $enabled, true, $checkbox);
+		return JHtml::_('jGrid.state', $states, $value, $i, 'banners.', $enabled, true, $checkbox);
```

Comment:
If we use the existing rules for JLoader where every capital letter is supposed to represent a new directory then that should let us deal with inconsistencies like with `JHtmlJGrid` or `JHtmlContentLanguage` which are wrong.